src/anemoi/utils/commands/metadata.py

```diff
@@ -0,0 +1,398 @@
+# granted to it by virtue of its status as an intergovernmental organisation
+# nor does it submit to any jurisdiction.
+
+
+import json
+import logging
+import os
+import shutil
+import subprocess
+from argparse import ArgumentParser
+from argparse import Namespace
+from tempfile import TemporaryDirectory
+from typing import Any
+from typing import Dict
+
+import yaml
+
+from . import Command
+
+LOG = logging.getLogger(__name__)
+
+EDITOR_OPTIONS = {"code": ["--wait"]}
+
+
+class Metadata(Command):
+    """Edit, remove, dump or load metadata from a checkpoint file."""
+
+    def add_arguments(self, command_parser: ArgumentParser) -> None:
+        """Add command line arguments to the parser.
+
+        Parameters
+        ----------
+        command_parser : ArgumentParser
+            The argument parser to which the arguments will be added.
+        """
+        from anemoi.utils.checkpoints import DEFAULT_NAME
+
+        command_parser.add_argument("path", help="Path to the checkpoint.")
+
+        group = command_parser.add_mutually_exclusive_group(required=False)
+        remove_metdata_group = command_parser.add_argument_group(title="Metadata removal")
+
+        group.add_argument(
+            "--dump",
+            action="store_true",
+            help=(
+                "Extract the metadata from the checkpoint and print it to the standard output"
+                " or the file specified by ``--output``, in JSON or YAML format."
+            ),
+        )
+        group.add_argument(
+            "--load",
+            action="store_true",
+            help=(
+                "Set the metadata in the checkpoint from the content"
+                " of a file specified by the ``--input`` argument."
+            ),
+        )
+
+        group.add_argument(
+            "--edit",
+            action="store_true",
+            help=(
+                "Edit the metadata in place, using the specified editor."
+                " See the ``--editor`` argument for more information."
+            ),
+        )
+
+        group.add_argument(
+            "--view",
+            action="store_true",
+            help=(
+                "View the metadata in place, using the specified pager."
+                " See the ``--pager`` argument for more information."
+            ),
+        )
+
+        remove_metdata_group.add_argument(
+            "--remove",
+            action="store_true",
+            help="Remove the metadata from the checkpoint.",
+        )
+
+        remove_metdata_group.add_argument(
+            "--inplace",
+            action="store_true",
+            help="If set, update the source file in place instead of writing to a separate target.",
+        )
+
+        remove_metdata_group.add_argument(
+            "--output-checkpoint",
+            type=str,
+            help="Path to checkpoint without metadata. Required unless --inplace is set.",
+        )
```

Comment:
This argument is never used. Should this be just `--output` instead of `--output-checkpoint` ?